Src/AutoFixture/Kernel/ParameterSpecification.cs

```diff
@@ -95,7 +95,9 @@ private bool ParameterIsCompatibleWithTargetType(object request)
 
         private bool ParameterHasTargetName(object request)
         {
-            return ((ParameterInfo)request).Name == this.targetName;
+            return ((ParameterInfo)request)
+                   .Name
+                   .Equals(this.targetName, StringComparison.OrdinalIgnoreCase);
```

Comment:
Isn't this a breaking change?

What if someone wishes to use this class, but with a case-sensitive string comparison?